src/app/profiles/multipleChoiceEditor.tsx

```diff
@@ -0,0 +1,197 @@
+'use client'
+
+import {Button} from "@/components/ui/button"
+import {
+    Card,
+    CardContent,
+    CardDescription,
+    CardFooter,
+    CardHeader,
+    CardTitle,
+} from "@/components/ui/card"
+import { Label } from "@/components/ui/label"
+import { RadioGroup, RadioGroupItem } from "@/components/ui/radio-group"
+import { Dispatch, SetStateAction, useRef, useState } from "react"
+import { SortableContext, verticalListSortingStrategy, useSortable, arrayMove } from "@dnd-kit/sortable"
+import { DndContext, DragEndEvent, closestCenter } from '@dnd-kit/core'
+import { CSS } from '@dnd-kit/utilities'
+import { Formik, Form, Field, FieldArray, FieldArrayRenderProps, FormikErrors } from 'formik'
+
+type Answer = {
+    id: number
+    label: string
+}
+
+const SortableInput = ({answer, index, answers, setAnswers, arrayHelpers, setFieldValue}: {answer: Answer, index: number, answers: Answer[], setAnswers: Dispatch<SetStateAction<Answer[]>>, arrayHelpers: FieldArrayRenderProps, setFieldValue: (field: string, value: any, shouldValidate?: boolean | undefined) => Promise<void | FormikErrors<{question: string; answers: string[];}>>}) => {
+    const {
+        attributes,
+        listeners,
+        setNodeRef,
+        transform,
+        transition,
+    } = useSortable({id: answer.id})
+
+    const style = {
+        transition,
+        transform: CSS.Transform.toString(transform),
+    }
+
+    return (
+        <div key={index} style={style} {...attributes}>
+            <br></br>
+            <Label>Answer {index + 1}</Label>
```

Comment:
nit;

UI-wise, I'm unconvinced that we need a label for each answer option here